lib/v2/sspai/index.js

```diff
@@ -9,7 +9,11 @@ module.exports = async (ctx) => {
     });
     const items = await Promise.all(
         resp.data.data.map((item) => {
-            const link = `https://sspai.com/api/v1/article/info/get?id=${item.id}&view=second`;
+            const link = `https://sspai.com/api/v1/${
+                item.slug && item.slug !== ''
```

Comment:
https://deepscan.io/dashboard/#view=project&tid=6244&pid=8135&bid=92448&subview=pull-request&prid=1693347

Reply:
Fixed it.